src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -41,6 +41,8 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-workspaces
 ----
 ====
 
+IMPORTANT: To run more than one workspace at a time when using this strategy together with the `common` PVC strategy, configure persistent volumes to use `ReadWriteMany` access mode.
```

Comment:
I'd add something along the lines of `Alternatively, limit the number of concurrently running workspaces per user to 1 when using _One namespace per user_ namespace strategy.`

The reason is that `ReadWriteMany` is not even available on some k8s installations or kinds of persistent volumes, so we should offer a solution that works around that limitation.

Reply:
> I'd add something along the lines of Alternatively, limit the number of concurrently running workspaces per user to 1

@metlos, I believe this is sufficiently implied by *To run more than one workspace at a time*.